src/ImageSharp/Memory/DiscontiguousBuffers/MemoryGroup{T}.cs

```diff
@@ -41,7 +41,7 @@ private MemoryGroup(int bufferLength, long totalLength)
     /// <inheritdoc />
     public bool IsValid { get; private set; } = true;
 
-    public MemoryGroupView<T> View { get; private set; }
+    public MemoryGroupView<T>? View { get; private set; }
```

Comment:
This should not be nullable. In what states is it `null`?

Reply:
I can dismiss this compiler error with null!; All the inheriting classes make sure in the constructor that View is set